tsl/test/expected/chunk_utils_internal.out

```diff
@@ -89,17 +117,35 @@ SELECT * from test1.hyper1 ORDER BY 1;
 
 DELETE FROM test1.hyper1 WHERE time = 20;
 DELETE FROM test1.hyper1 WHERE temp = 0.5;
-ERROR:  Delete not permitted on frozen chunk "_hyper_1_1_chunk" 
+ERROR:  unable to modify frozen chunk _hyper_1_1_chunks
 SELECT * from test1.hyper1 ORDER BY 1;
  time | temp 
 ------+------
    10 |  0.5
    30 |  0.5
 (2 rows)
 
+-- TEST delete on unfrozen chunk should be possible
+UPDATE test1.hyper1 SET temp = 40 WHERE time = 20;
+-- Test with cast (chunk path pruning can not be done during query planning)
+UPDATE test1.hyper1 SET temp = 40 WHERE time = 20::text::float;
+-- TEST delete on unfrozen chunk should be possible
```

Comment:
Could you add this? SELECT * FROM test1.hyper1 WHERE time = 20;

Reply:
I have added some selects to the tests to give a better understanding what is currently stored in the test table.